content/applications/productivity/whatsapp.rst

```diff
@@ -7,17 +7,786 @@ content. Businesses can use `WhatsApp Business
 <https://developers.facebook.com/products/whatsapp/>`_ to communicate with their customers by text,
 send documents and provide support.
 
-With the **Odoo WhatsApp** app, you can connect your WhatsApp Business Account (WABA) to your Odoo
-database, which allows you to:
+With the **Odoo WhatsApp** app, a company can connect a WhatsApp Business Account (WABA) to an Odoo
+database, which allows for the following:
 
-- Receive and reply to WhatsApp messages directly from your Odoo database
-- Send pre-approved templates that use dynamic values, such as:
+- Receive and reply to WhatsApp messages directly from a Odoo database
+- Create new templates with dynamic placeholders/variables
+- Send pre-approved templates that use dynamic variables, such as:
 
   - Quotations from the Sales app
   - Receipts and invoices from the Point of Sale app
   - Tickets from the Events app
 
 .. seealso::
-   - `Meta Business: create message templates for your WhatsApp Business account <https://www.facebook.com/business/help/2055875911147364>`_
-   - `Meta Business: connect your phone number to your WhatsApp Business account <https://www.facebook.com/business/help/456220311516626>`_
-   - `Meta Business: change your WhatsApp Business display name <https://www.facebook.com/business/help/378834799515077>`_
+   - `Meta Business: create message templates for the WhatsApp Business account <https://www.facebook.com/business/help/2055875911147364>`_
+   - `Meta Business: connect a phone number to the WhatsApp Business account <https://www.facebook.com/business/help/456220311516626>`_
+   - `Meta Business: change the WhatsApp Business display name <https://www.facebook.com/business/help/378834799515077>`_
+
+WhatsApp is a messaging service operated by Meta, which is the parent company of Facebook. Whatsapp
+is commonly used as a communication tool in many countries and by many businesses. This
+documentation will cover the integration of a WhatsApp Business Account with Odoo. The company's
+Meta account is configured in Odoo via a :abbr:`API (Application Programming Interface)` connection.
+
+The Whatsapp connector supports two flows: company initiated, and customer initiated. A company can
+initiate a discussion by sending a template to one or a batch of people. Once the template is sent,
+the recipient can answer to trigger a discussion between the sender and the receiver (a discuss chat
+window will popup if the customer answers within 15 days).
```

Comment:
wording
- cause>effect
- popup is a noun; pop up is the action you're looking for 😉 

*Discuss* is capitalized and italicized here since it's an Odoo product, no?

```suggestion
recipient can answer in order to trigger a discussion between the sender and the receiver (a
*Discuss* chat window will pop up if the customer answers within 15 days).
```